include/boost/geometry/algorithms/detail/distance/segment_to_box.hpp

```diff
@@ -425,29 +437,35 @@ class segment_to_box_2D
                                        SBStrategy const& sb_strategy)
         {
             boost::ignore_unused(sb_strategy);
-            typedef cast_to_result<ReturnType> cast;
+
             LessEqual less_equal;
 
             // p0 is above the upper segment of the box (and inside its band)
             // then compute the vertical (i.e. meridian for spherical) distance
             if (less_equal(geometry::get<0>(top_left), geometry::get<0>(p_max)))
             {
                 ReturnType diff =
-                sb_strategy.get_distance_ps_strategy().vertical_or_meridian(
-                                    geometry::get_as_radian<1>(p_max),
-                                    geometry::get_as_radian<1>(top_left));
+                    sb_strategy.get_distance_ps_strategy().vertical_or_meridian
+                    (
+                        geometry::get_as_radian<1>(p_max),
+                        geometry::get_as_radian<1>(top_left),
+                        geometry::get_as_radian<0>(p0)
+                    );
```

Comment:
The formatting looks unusual. Do we have guidelines regarding the parentheses?

Reply:
no only this 
```
Parentheses around expressions should not be pre/post-fixed with spaces.
```
I followed the bracket case.

Reply:
is this better?
```
ReturnType diff =
    sb_strategy.get_distance_ps_strategy().vertical_or_meridian(
        geometry::get_as_radian<1>(p_max),
        geometry::get_as_radian<1>(top_left),
        geometry::get_as_radian<0>(p0));
```